src/kokkos/test/kokkos/OneHistoContainer_t.cc

```diff
@@ -65,14 +65,16 @@ void go() {
         });
 
     Kokkos::deep_copy(KokkosExecSpace(), histo_h, histo_d);
-    assert(histo_h().off[-1] == 0);
+    // TODO: fails to compile with GCC, is this comparison really needed?
+    //assert(histo_h().off[-1] == 0);
```

Comment:
@jtchilders Can this assert be removed?

Reply:
I presume it was in the original which is why I left it. My guess is that it is a sanity check and can be removed.

Reply:
Thanks, removed.